commit 55e9ae2d3f06d1c07c612b1fced1e419bc37132d (HEAD -> refs/heads/develop)
Author: Ben Webb <ben@salilab.org>
Date:   Mon Jun 2 12:07:36 2025 -0700

    Use sysconfig instead of distutils
    
    distutils has been removed as of Python 3.12,
    so use the replacement 'sysconfig' module.

diff --git a/tools/__init__.py b/tools/__init__.py
index 79ebecb..c744b5f 100644
--- a/tools/__init__.py
+++ b/tools/__init__.py
@@ -104,8 +104,12 @@ def _get_python_include(env):
     elif env['wine']:
         return '/usr/lib/w32comp/w32python/2.7/include/'
     else:
-        import distutils.sysconfig
-        return distutils.sysconfig.get_python_inc()
+        try:
+            import sysconfig
+            return sysconfig.get_path('include')
+        except ImportError:
+            import distutils.sysconfig
+            return distutils.sysconfig.get_python_inc()
 
 def _add_release_flags(env):
     """Add compiler flags for release builds, if requested"""
@@ -522,7 +526,10 @@ def get_pyext_environment(env, mod_prefix, cplusplus=False):
             # Make sure compilers are in the PATH, so that Python's script for
             # building AIX extension modules can find them:
             e['ENV']['PATH'] += ':/usr/vac/bin'
-        from distutils.sysconfig import get_config_vars
+        try:
+            from sysconfig import get_config_vars
+        except ImportError:
+            from distutils.sysconfig import get_config_vars
         vars = get_config_vars('CC', 'CXX', 'OPT', 'BASECFLAGS', 'LDSHARED',
                                'SO', 'EXT_SUFFIX')
         (cc, cxx, opt, basecflags, ldshared, so, ext_suffix) = vars
